docs/build/program/apis/_index.md

```diff
@@ -173,7 +173,7 @@ The [navigation service](/mobility/navigation/) supports the following methods:
 
 ### SLAM
 
-The {{< glossary_tooltip term_id="slam" text="Simultaneous Localization And Mapping (SLAM) service" >}} supports the following methods:
+The [{{< glossary_tooltip term_id="slam" text="Simultaneous Localization And Mapping (SLAM) service " >}}](/mobility/slam/) supports the following methods:
```

Comment:
We either use the tooltip or the direct link. I think the direct link is correct here
```suggestion
The [Simultaneous Localization And Mapping (SLAM) service](/mobility/slam/) supports the following methods:
```